VirtoCommerce.Storefront/Domain/Countries/FileSystemCountriesService.cs

```diff
@@ -35,20 +34,15 @@ public async Task<IList<Country>> GetCountriesAsync()
             var cacheKey = CacheKey.With(GetType(), "GetCountries");
             return await _memoryCache.GetOrCreateAsync(cacheKey, async (cacheEntry) =>
             {
-                List<Country> result = new List<Country>();
-
-                var regions = CultureInfo.GetCultures(CultureTypes.SpecificCultures)
-                    .Select(GetRegionInfo)
-                    .Where(r => r != null)
-                    .ToList();
+                var result = new List<Country>();
```

Comment:
Seems you can remove property `GetRegionInfo` too, if it is not used

Reply:
removed